net/wget2/Portfile

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                wget2
+version             2.1.0
+revision            0
+
+homepage            https://gitlab.com/gnuwget/wget2
+
+description         \
+    GNU Wget2 is the successor of GNU Wget, a file and recursive website \
+    downloader.
+
+long_description    {*}${description}.
+
+categories          net www
+installs_libs       no
+license             GPL-3+
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+master_sites        https://ftp.gnu.org/gnu/wget
+
+checksums           rmd160  3e39a932a90eaaff69e90ecb761a4946e3444e10 \
+                    sha256  a05dc5191c6bad9313fd6db2777a78f5527ba4774f665d5d69f5a7461b49e2e7 \
+                    size    3867884
+
+depends_build-append \
+                    path:bin/pkg-config:pkgconfig
+
+depends_lib-append  path:lib/pkgconfig/gnutls.pc:gnutls \
+                    port:brotli \
+                    port:bzip2 \
+                    port:gpgme \
+                    port:libhsts \
+                    port:libidn2 \
+                    port:libmicrohttpd \
+                    port:libpsl \
+                    port:lzma \
+                    port:nghttp2 \
+                    port:pcre2 \
+                    port:zlib \
+                    port:zstd
+
```

Comment:
@herbygillot If patching is not in the plans, then adding something like this will fix it:


```
compiler.blacklist-append \
                    {*gcc-[34].*}
```

Reply:
Added